include/inviwo/core/datastructures/light/directionallight.h

```diff
@@ -49,7 +47,7 @@ class DirectionalLight : public LightSource {
      * @return Radiant flux in watt.
      */
     virtual vec3 getPower() const override {
-        return getIntensity() * getArea() * static_cast<float>(M_PI);
+        return getIntensity() * getArea() * glm::two_pi<float>();
```

Comment:
two pi? Why is the area weighted by pi in the first place...

Reply:
Glm::pi()
As for pi, see the pbrt book